tests/read_test.h

```diff
@@ -323,6 +326,32 @@ void processEvent(podio::EventStore& store, int eventNum) {
   check_fixed_width_value(arbComps.fixedInteger64, std::int64_t{-1234567890123456789}, "int64");
   check_fixed_width_value(arbComps.fixedInteger32, std::int32_t{-1234567890}, "int32");
   check_fixed_width_value(arbComps.fixedUnsigned16, std::uint16_t{12345}, "uint16");
+
+
+
+  auto& usrInts = store.get<podio::UserDataCollection<int> >("userInts");
+
+#if 0 //access via vector
```

Comment:
Are these left over from testing, or are they actually required?

Reply:
Sort of: testing - and as example how the underlying vector could be accessed, if needed. Can also remove eventually.
All this is still very much WIP ...